packages/arb-avm-cpp/data_storage/src/arbcore.cpp

```diff
@@ -1319,26 +1313,6 @@ void ArbCore::operator()() {
                     combined_machine_cache.timed_add(
                         std::make_unique<Machine>(*core_machine));
 
-                    checkpoint_pruning_age_timestamp =
-                        seconds_since_epoch() -
-                        coreConfig.checkpoint_pruning_age_seconds;
-                    {
-                        std::lock_guard<std::mutex> lock(
-                            checkpoint_pruning_mutex);
-                        checkpoint_pruning_gas_used =
-                            unsafe_checkpoint_pruning_gas_used;
-                    }
-
-                    if (coreConfig.checkpoint_pruning_age_seconds > 0) {
-                        // Do actual pruning later
-                        perform_age_pruning = true;
-                    }
-
-                    if (checkpoint_pruning_gas_used > 0) {
-                        // Do actual pruning later
-                        perform_gas_pruning = true;
-                    }
-
```

Comment:
I think we need to set `perform_pruning` to true here

Reply:
:+1: